src/roles/check_database_connection/tasks/main.yaml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Check Foreman database connectivity and credentials
+  community.postgresql.postgresql_ping:
```

Comment:
Does it mean that we need to make sure this module is present on the machine when the command runs?

Reply:
Yes and `community.postgresql` collection is already included in our [src/requirements.yml](https://github.com/theforeman/foremanctl/blob/master/src/requirements.yml#L2), so it should be available when the checks run.